frontend/app/templates/components/user-component/user-guide.hbs

```diff
@@ -5,7 +5,7 @@
     </div>
     <div class="ui segments">
         <div class="ui segment">
-            <div class="ui raised segment">
+            <div class="ui">
```

Comment:
Can be changed to 
```html
<div class='ui basic segment'>
</div>
```

Reply:
with basic segment
![shif](https://user-images.githubusercontent.com/25414121/42958678-e45fdfc2-8ba3-11e8-9f7f-4ee773a29c10.png)


with this PR
![guuide](https://user-images.githubusercontent.com/25414121/42958683-ea8a489c-8ba3-11e8-8330-bcc4f282a248.png)


the current changes looks more even than the `basic segment` would look. you can see that the titles are a bit shifted towards right.

Reply:
Fine then